internal/operators/lvm/config.go

```diff
@@ -7,13 +7,14 @@ import (
 const (
 	// LvmMinOpenshiftVersion is the minimum OCP version in which lvmo is supported
 	// Any changes here should be updated at line 16 too.
-	LvmMinOpenshiftVersion string = "4.12.0-0.0"
+	LvmMinOpenshiftVersion  string = "4.11.0"
+	lvmsMinOpenshiftVersion string = "4.12.0"
 )
 
 type Config struct {
 	LvmCPUPerHost          int64  `envconfig:"LVM_CPU_PER_HOST" default:"1"`
 	LvmMemoryPerHostMiB    int64  `envconfig:"LVM_MEMORY_PER_HOST_MIB" default:"1200"`
-	LvmMinOpenshiftVersion string `envconfig:"LVM_MIN_OPENSHIFT_VERSION" default:"4.12.0-0.0"`
+	LvmMinOpenshiftVersion string `envconfig:"LVM_MIN_OPENSHIFT_VERSION" default:"4.11.0"`
```

Comment:
Why we need LvmMinOpenshiftVersion twice - here and in the const?

Reply:
we use this const in other pkg like cnv, and we need to get min openshift for lvmo/s support and there will case when this struct is not filled with default values and we are trying to get the  min openshift for lvmo/s.